assignments/prototypes.js

```diff
@@ -102,9 +136,142 @@
+
+
+  function Anathemanoid(anatha) {
+    CharacterStats.call(this, anatha);
+    this.team = anatha.team;
+    this.weapons = anatha.weapons;
+    this.language = anatha.language; 
+    this.darknessbuff = 10;
+  }
+
+  Anathemanoid.prototype = Object.create(CharacterStats.prototype);
+
+  var villanweaponsarray = ['Division Strikes',
+  'Force Field',
+  'Slimebow',];
+
+  const Villan = new Anathemanoid({
+    createdAt: new Date(),
+    dimensions: {
+      length: 4,
+      width: 2,
+      height: 6,
+    },
+    healthPoints: 60,
+    name: 'Xap',
+    team: 'pride',
+    weapons: villanweaponsarray,
+    language: 'Uncommon Tongue',
+  });
+
+  var heroweaponsarray = ['Broadsword',
+  'Concentrated Energy',
+  'Sunchucks',];
+
+  const Hero = new Humanoid({
+    createdAt: new Date(),
+    dimensions: {
+      length: 3,
+      width: 3,
+      height: 4,
+    },
+    healthPoints: 70,
+    name: 'Pax',
+    team: 'Good People',
+    weapons: heroweaponsarray,
+    lightbuff: 5,
+    language: 'Humanisheseic',
+  });
+
+  var ttlweapons = heroweaponsarray.length;
+
+  //FIGHT!
+
+  function getRandomInt(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+  }  
+
+    var heropts = 75;
+    var villpts = 70;
+
+    console.log(heropts);
+
+
+    console.log('BATTLE TIME!');
+
+    var weapttl = ttlweapons;
+    var turn = getRandomInt(2);
+    var weap = getRandomInt(weapttl);
+
+    function herobattle() {
+      console.log("Hero's turn!");
+      var ttldmg = 0;
+      ttldmg = getRandomInt(11);
+      weap = getRandomInt(weapttl);
+      console.log(`${Hero.name} hit the enemy with ${Hero.weapons[weap]} for ` + ttldmg + `dmg`);
+      villpts -= ttldmg;
+      turn += 1;
+    }
+
+    function villanbattle() {
+      console.log("Villan's turn!");
+      var ttldmg = 0;
+      ttldmg = getRandomInt(11);
+      weap = getRandomInt(weapttl);
+      console.log(`${Villan.name} hit the enemy with ${Villan.weapons[weap]} for ` + ttldmg + `dmg`);
+      heropts -= ttldmg;
+      turn -= 1;
+    }
+
+  do {
+    if (turn === 0 && heropts > 0) {
+      console.log("Hero's turn!");
+      var ttldmg = 0;
+      ttldmg = getRandomInt(11);
+      weap = getRandomInt(weapttl);
+      console.log(`${Hero.name} hit the enemy with ${Hero.weapons[weap]} for ` + ttldmg + `dmg`);
+      document.getElementById('hbo').innerHTML = `${Hero.name} hit the enemy with ${Hero.weapons[weap]} for ` + ttldmg + `dmg`;
+      villpts -= ttldmg;
+      turn += 1;
+      continue;
+    } else if (turn === 1 && villpts > 0) {
+      console.log("Villan's turn!");
+      var ttldmg = 0;
+      ttldmg = getRandomInt(11);
+      weap = getRandomInt(weapttl);
+      console.log(`${Villan.name} hit the enemy with ${Villan.weapons[weap]} for ` + ttldmg + `dmg`);
+      document.getElementById('vbo').innerHTML = `${Villan.name} hit the enemy with ${Villan.weapons[weap]} for ` + ttldmg + `dmg`;
```

Comment:
Great job using DOM manipulation to allow for updates to the battle to be viewed.